concordium-sdk/src/main/java/com/concordium/sdk/requests/dumpstart/DumpRequest.java

```diff
@@ -0,0 +1,26 @@
+package com.concordium.sdk.requests.dumpstart;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.io.File;
+
+/**
+ * The dump request used when requesting dumping of packages.
+ * See {@link com.concordium.sdk.ClientV2#dumpStart(DumpRequest)}
+ */
+@Builder
+@Getter
+public class DumpRequest {
+
+    /**
+     * Which file to dump the packages into
+     * Requires a valid path
+     */
+    private final File file;
```

Comment:
I think a `Path` would be the better option here, as the SDK retains no handle to the file that is specified.